tools/utilAppRunner.js

```diff
@@ -14,7 +14,7 @@ function runUtilApp (cmd, file, stdioOptions) {
   }
   cmd = cmd.split(' ')
   if (process.env.NODE_ENV === 'development') {
-    cmd.push('--user-data-dir=brave-development')
+    cmd.push('--user-data-dir=../../../brave-development')
```

Comment:
Good question, @mrose17. This change ensures Brave has the right path to `ledger-state.json` when `NODE_ENV` is set to `development`.

Reason:
For these utility scripts, the userData path is retrieved via `app.getPath(\"userData\")` run in `./tools/lib/utilApp/index.js` (three directory levels removed from the base of the repo).
Before this change, `./tools/utilAppRunner.js` was running brave with `--user-data-dir=brave-development`, which would cause Brave to look in `./tools/lib/utilApp/brave-development` instead of `./brave-development`.

What tests do you think it might break?